general/server/storage.md

```diff
@@ -0,0 +1,30 @@
+---
+uid: server-storage
+title: Storage
+---
+
+## Storage
+
+Jellyfin is designed to directly read media from the filesystem. This means to pass a network storage device that is using samba or NFS must be directly mounted to the OS. The Jellyfin database also should be stored locally and not on a network storage device.
+
+## Docker/VM's
+
+For storage, a library database can take up to 50 to 100 GB. The [transcoding](https://github.com/jellyfin/jellyfin-docs/blob/master/general/server/transcoding.md) folder needs roughly the same size as the original media if it's being transcoded at the same bitrate. A single 50GB Blu-Ray Remux by itself can take up to approximately 60GB or as little as 15GB, depending on the quality selected. If the transcoding folder is held on the same storage as the database, this must be taken into consideration.
```

Comment:
That's ..... a very very large database.  I'm currently sitting at 4.2G between the config and cache directories in my docker container.  It's probably worth noting that this is going to vary greatly depending on how much media you have.  Telling users that they're going to need 50+ gigs for a database is a bit intimidating.

Might want to make transcoding a separate paragraph at least.  Also, you want to link to another docs page, not the raw Github document.

Reply:
So I'm sitting at 6.5GB but disk space is sitting at 80GB.